src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.cs

```diff
@@ -33,5 +33,12 @@ public void VaultCRUDTests()
         {
             TestController.NewInstance.RunPsTest("Test-RecoveryServicesVaultCRUDTests");
         }
+
+        [Fact(Skip = "TODO: only works for live mode")]
```

Comment:
Do not add a skipped test.  Tests marked as 'live only' will not be run in any CI job.  Please remove the skip